repos/runtime.common.props

```diff
@@ -0,0 +1,145 @@
+
+      <BuildCommand>$(ProjectDirectory)build-source-build$(ShellExtension) $(BuildArguments) $(FlagParameterPrefix)coreclr-args $(CoreClrBuildArguments) $(FlagParameterPrefix)libraries-args $(LibrariesBuildArguments) $(FlagParameterPrefix)installer-args $(InstallerBuildArguments)</BuildCommand>
+      <BuildCommand Condition="$(Platform.Contains('arm')) AND '$(BuildArchitecture)' != 'arm64'">$(ArmEnvironmentVariables) $(BuildCommand)</BuildCommand>
+      <CleanCommand>$(ProjectDirectory)/clean$(ShellExtension)</CleanCommand>
+  </PropertyGroup>
+
+  <!-- Output / source-build flags -->
+  <PropertyGroup>
+      <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
+      <PackagesOutput>$(ProjectDirectory)bin/Product/$(TargetOS).$(Platform).$(Configuration)/.nuget/pkg</PackagesOutput>
+      <ShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</ShippingPackagesOutput>
+      <NonShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/NonShipping/</NonShippingPackagesOutput>
+      <NuGetConfigFile>$(ProjectDirectory)/NuGet.config</NuGetConfigFile>
+      <DependencyVersionInputRepoApiImplemented>true</DependencyVersionInputRepoApiImplemented>
+      
+      <!-- core-setup does not destabilize for intermediate servicing builds -->
+      <IsStable>true</IsStable>
+  </PropertyGroup>
+
+  <!-- Environment Restore Sources -->
+  <PropertyGroup>
+      <EnvironmentRestoreSources>$(SourceBuiltPackagesPath)</EnvironmentRestoreSources>
+      <EnvironmentRestoreSources>$(EnvironmentRestoreSources)%3B$(ShippingPackagesOutput)</EnvironmentRestoreSources>
+      <EnvironmentRestoreSources>$(EnvironmentRestoreSources)%3B$(NonShippingPackagesOutput)</EnvironmentRestoreSources>
+      <EnvironmentRestoreSources Condition="'$(OfflineBuild)' == 'true'">$(EnvironmentRestoreSources)%3B$(ReferencePackagesDir)%3B$(PrebuiltPackagesPath)</EnvironmentRestoreSources>
+      <EnvironmentRestoreSources Condition="'$(OfflineBuild)' != 'true'">$(EnvironmentRestoreSources)%3Bhttps://api.nuget.org/v3/index.json</EnvironmentRestoreSources>
+      <EnvironmentRestoreSources Condition="'$(OfflineBuild)' != 'true'">$(EnvironmentRestoreSources)%3Bhttps://dotnetfeed.blob.core.windows.net/dotnet-core/index.json</EnvironmentRestoreSources>
+      <EnvironmentRestoreSources Condition="'$(OfflineBuild)' != 'true'">$(EnvironmentRestoreSources)%3Bhttps://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json</EnvironmentRestoreSources>
+
+  </PropertyGroup>
+
+  <!-- SDK Overrides -->
+  <ItemGroup>
+      <UseSourceBuiltSdkOverride Include="@(ArcadeSdkOverride)" />
+      <UseSourceBuiltSdkOverride Include="@(ArcadeCoreFxTestingOverride)" />
+      <UseSourceBuiltSdkOverride Include="@(ArcadePackagingOverride)" />
+      <!--UseSourceBuiltSdkOverride Include="@(ILSdkOverride)" />
+      <BuiltSdkPackageOverride Include="@(ILSdkOverride)" /-->
```

Comment:
Since the IL SDK is built and used by dotnet/runtime, this won't be coming back--can delete.

Reply:
Deleted